src/panels/Home.module.css

```diff
@@ -85,16 +85,18 @@
 
 .left, .right
 {
-  margin: 0px var(--lr_padding);
+  margin: px var(--lr_padding);
```

Comment:
This looks like an error

Reply:
Oh, yea sorry this was an accidental delete.

Reply:
That's fine, it doesn't change anything.  The rest is good, I'm glad you figured everything out